resources/content/config-content/haproxy/apply.sh

```diff
@@ -9,9 +9,15 @@ awk '/BEGIN.*PRIVATE KEY/{i++}{print > "/etc/haproxy/certs/server"i".pem"}' /etc
 rm /etc/haproxy/certs/certs.pem
 if [ ! -s /etc/haproxy/certs/default.pem ]; then rm /etc/haproxy/certs/default.pem;fi
 
-if ! haproxy -v | grep -q 'version 1.5'; then
-    echo deb http://archive.ubuntu.com/ubuntu trusty-backports main universe > /etc/apt/sources.list.d/backports.list
-    apt-get update && apt-get install -y haproxy -t trusty-backports
+
+
+if ! haproxy -v | grep -q 'version 1.6'; then
+    # save config file as update to haproxy 1.6 overrides it
+    cp /etc/haproxy/haproxy.cfg /etc/haproxy/haproxy.cfg_tmp
+    apt-get update -y && apt-get install -y software-properties-common \
+    && add-apt-repository -y ppa:vbernat/haproxy-1.6 \
+    && apt-get update -y && apt-get install -y haproxy
```

Comment:
@ibuildthecloud by some reason, I see the last apt-get being stuck on the lb instance launch. Not sure why it happens because when I test this command manually on one of my pre-upgraded haproxy instances, it works fine.

Also when I remove "apt-get update -y" from the line 17, it also works fine. So seems to happen only when 2 apt-get updates are present (one before getting ppa:vbernat/haproxy-1.6, and one after)